api/api_test.go

```diff
@@ -165,21 +169,22 @@ func hasRoute(r *mux.Router, path, method string) bool {
 
 func apiSetup() (*API, *httptest.ResponseRecorder, *mock.MockCognitoIdentityProviderClient) {
 	var (
-		ctx                                                 = context.Background()
-		r                                                   = mux.NewRouter()
-		poolID, clientID, clientSecret, awsRegion, authFlow = "us-west-11_bxushuds", "client-aaa-bbb", "secret-ccc-ddd", "eu-west-1234", "USER_PASSWORD_AUTH"
-		allowedDomains                                      = []string{"@ons.gov.uk", "@ext.ons.gov.uk"}
+		ctx                                       = context.Background()
+		r                                         = mux.NewRouter()
+		poolID, clientID, clientSecret, awsRegion = "us-west-11_bxushuds", "client-aaa-bbb", "secret-ccc-ddd", "eu-west-1234"
+		authFlow                                  = types.AuthFlowTypeUserPasswordAuth // "USER_PASSWORD_AUTH"
```

Comment:
```suggestion
		authFlow                                  = types.AuthFlowTypeUserPasswordAuth
```
Don't think comment is warranted.

Reply:
👍 - now deleted